database/common/common.go

```diff
@@ -0,0 +1,9 @@
+package common
```

Comment:
we are missing a copyright header here as well.

Reply:
I'll add that.